docroot/profiles/custom/cgov_site/modules/custom/cgov_image/src/CgovImageGetThemeImages.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\cgov_image;
+
+use Drupal\image\Entity\ImageStyle;
+use Drupal\cgov_image\CgovImageTools;
+
+/**
+ * Class CgovImageGetThemeImages.
+ */
+class CgovImageGetThemeImages extends \Twig_Extension
+{
+  /**
+   * A list with all the defined functions. The first parameter is the name
+   * by which we will call the function. The second parameter is the
+   * name of the function that will implement the function.
+   */
+  public function getFunctions() {
+    return [
+      new \Twig_SimpleFunction('get_theme_images', [$this, 'getThemeImages']),
+    ];
+  }
+
+  /**
+   * Returns the extension name.
+   */
+  public function getName() {
+    return 'cgov_image.CgovImageGetThemeImages';
+  }
+
+  /**
+   * Function that returns an example text with a color passed as a parameter.
+   */
+  public function getThemeImages($image_style) {
+    $test = new CgovImageTools();
+    $crop = $test->findCropByStyle($image_style);
+    if ($crop === "4x3") {
+      $uri = 'module://cgov_image/img/placeholder-4x3.png';
```

Comment:
Use the `$crop` variable to get the file name so this thing scales. I.E., no switch statement.